lib/internal/repl/utils.js

```diff
@@ -413,9 +413,9 @@ function setupPreview(repl, contextSymbol, bufferSymbol, active) {
       // Do not preview `undefined` if colors are deactivated or explicitly
       // requested.
       if (inspected === 'undefined' &&
-          (!repl.useColors || repl.ignoreUndefined)) {
-        return;
-      }
+      (!repl.useColors || repl.ignoreUndefined)) {
+            return;
+        }
```

Comment:
Can you please revert the unrelated changes please?
```suggestion
          (!repl.useColors || repl.ignoreUndefined)) {
        return;
      }
```